docs/logical_data_model.md

```diff
@@ -163,7 +163,8 @@ class ActivityReport {
   ttaType : array<string>
   context : string
   pageState : json
-  managerNotes : string
+  legacyManagerNotes : string
+  legacyApprovingManagerId : integer(32) REFERENCES public.Users.id
```

Comment:
`approvingManagerId` should have been in this schema already. I'm proposing renaming this field `legacyApprovingManagerId`.